### DISCUSSION TITLE
Merge Projects

### DISCUSSION THREAD

**Discussion by pwbrown44** [vkbo/novelWriter#794] [Q&A]
Is there a way to merge projects into one master project?

**Comment by vkbo** [comment]
There is no direct way at the moment. I have on my to do list to add a tool to import from another project.

There are two ways of importing the content itself. They both require a bit of cleanup afterwards, but it will get the job done.

Export the stuff you want to bring over to the other project from the Build tool into an `.nwd` file. Then you can make a new document in the other project and import the entire content of the exported file into the new file. There's an import feature in the Documents menu. You can then split up the imported document into individual files again with the split tool, also in the Documents menu. This works well for novel files, but your notes will have to be changed back to notes and moved to their correct notes folder.

Alternatively, you can just copy the files in the `contents` folder where your project files are stored on your harddrive of one project over to the other. When you open the project again the foreign files will be discovered and added to the project. However, since their parent folders don't exist in the other project, the files will end up at the bottom of each root folder where they were located in the other project. Make sure the root folders you had in the other project also exist in the project you copy the files to. This will just make it easier to clean up afterwards if there are many files. The discovered files will be labelled with "Recovered", so you can easily spot them.

**Note:** In version 1.3.1 there is a bug in the split tool if the last line of the document doesn't have a line ending after it. That line will then be lost in the split. You can fix it by making sure the imported document ends with a line break. I just discovered this bug and I'll fix it right away.
